### ISSUE TITLE
Merge norns-image repo into this repo

### ISSUE THREAD

**Issue by simonvanderveldt** [monome/norns#523]
We're making live harder for everyone by having two repos for the base system related items.
People who pull from git get broken systems, we get questions from people who are running norns on rpi because stuff breaks, etc.

Having a repo for the little bit of stuff that lives norns-image doesn't really make sense. Everything that's in there can easily live in this repo. So my suggestion would be to merge norns-image into this repo.

Suggestion for how to handle this:
`config` and `scripts` can stay directories, though I'm not sure all scripts in `scripts` are still relevant.
The readme's can either be merged with existing readme's or simply added.

**Comment by nf** [comment]
Another point in favor of doing this: When I was debugging some pretty useful information was hidden away in the norns-image readme files; would have been easier to find if it were in the norns repo.

**Comment by tehn** [comment]
while i like the idea of encouraging different platforms, the native hardware is certainly the focus. so should we just move the norns-image tree into a folder, say `system`?

i think @artfwo had some earlier reservations about this?

**Comment by artfwo** [comment]
If we're duplicating things between repos, then we should de-duplicate them on a case by case basis. I am not convinced that 2 repos serve the same purpose: norns is code for the norns "app", norns-image is system-specific stuff. If at some point we move to another distro, anything system-related (notifications, PRs, issues) would be easier to track in a different repo. @nf you point could be also valid, if we had the 2 repos merged (read: information was hidden in system/readme files).

In other words, merging won't harm a lot, but it's pointless.

**Comment by tehn** [comment]
this does bring up a good point.

provided norns-setup.md stays up to date (ie, specifies supernova etc) there's possibly not a strong reason to merge the two.

what's probably most important is more communication when we introduce a system-requirement change, ie, install supernova. it was sloppy of me to merge the norns repo without explicitly alerting the need for a supernova install--- i apologize.

**Comment by simonvanderveldt** [comment]
It was mainly about needing to change things in norns-image when we change things here (i.e. the wrapper arguments for example).

> If at some point we move to another distro, anything system-related (notifications, PRs, issues) would be easier to track in a different repo.

This seems rather theoretical, so not really relevant.

I don't see a disadvantage to merging so would like to go ahead and create a PR. I'm fine with a `system` dir if that helps?

**Comment by artfwo** [comment]
@simonvanderveldt we'd still have to change wrapper arguments in the merged subdirectory, so that doesn't solve the problem. also, what happens to norns-image commit history after the merge?

**Comment by simonvanderveldt** [comment]
> we'd still have to change wrapper arguments in the merged subdirectory, so that doesn't solve the problem

But it does, in that you can and should update those things in the same PR :)

> also, what happens to norns-image commit history after the merge?

We can keep the history by moving the stuff that's in the norns-image repo to a directory, merging it into a branch on this repo and creating a PR for that

**Comment by simonvanderveldt** [comment]
> Well, the alternative solution for the specific case about wrapper arguments would be to provide launcher scripts for crone and matron in norns.

I prefer simplifying to solve issues over adding intermediate layers

> This is not a systematic problem that is solved by merging repos, e.g. we also have some things outside of both repos (debian packages, update scripts) which we change sometimes.

Yeah, those items are also an issue which should be addressed

**Comment by simonvanderveldt** [comment]
Closing this for now, not really a problem anymore and we might end up with a different distro in the future after all :)
